packages/ui/src/data/requestArgs/index.ts

```diff
@@ -78,7 +86,7 @@ export const useRequestArgs = (): UseRequestArgsReturn => {
           operator: '$eq',
         },
       ],
-      populate: ['users.role', 'platforms'],
+      populate: ['users.role', 'platforms', 'jobs', 'role'],
```

Comment:
If we will remove `platforms` relation from profile, we can still reach the platform data with nested populate

```suggestion
      populate: ['users.role', 'jobs.platform', 'role'],
```